shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -144,6 +144,17 @@ public InitResult call() {
 
             System.loadLibrary("flutter");
 
+            // Pre-warm the default font manager as soon as possible on a background thread.
+            // It helps to reduce time cost of engine setup that blocks the platform thread.
+            new Thread(
```

Comment:
Use `Executors.newSingleThreadExecutor().submit()` for consistency with other threads used in `FlutterLoader`

Reply:
done